media/js/base/sentry.es6.js

```diff
@@ -16,32 +25,18 @@ if (
         .getElementsByTagName('html')[0]
         .getAttribute('data-sentry-dsn');
 
-    // Configure Sentry SDK
-    if (sentryDsn) {
-        init({
-            dsn: sentryDsn,
-            sampleRate: 0.1,
-            ignoreErrors: [
-                "SecurityError: Failed to execute 'open' on 'XMLHttpRequest'", // issue 10683
-                'NetworkError when attempting to fetch resource', // issue 10683
-                'Non-Error promise rejection captured with value' // issue 10380
-            ],
-            allowUrls: ['/media/js/', 'https://cdn-3.convertexperiments.com/'],
-            beforeSend(event) {
-                try {
-                    // https://github.com/getsentry/sentry-javascript/issues/3147
-                    if (
-                        event.exception.values[0].stacktrace.frames[0]
-                            .filename === '<anonymous>'
-                    ) {
-                        return null;
-                    }
-                } catch (e) {
-                    // do nothing.
-                }
+    const client = new BrowserClient({
```

Comment:
**Question**: Does the `BrowserClient` already ignore the errors that we listed inside of `ignoreErrors`? I just see a bunch of configurations that we had previously that we aren't keeping around

Reply:
Yeah the `InboundFilters` integration is what we used to ignore errors previously. However much of what we had in here was very old, and we have much more sensible alerting rules set up in the Sentry admin today, so I don't think the errors we were ignoring were really achieving much. We can always add this back in later if we need, but my hunch is we won't see much of a difference without the`InboundFilters` integration.